lib/Arch/Sleigh/PPCArch.cpp

```diff
@@ -0,0 +1,272 @@
+/*
+ * Copyright (c) 2022-present Trail of Bits, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include "PPC.h"
+
+#define INCLUDED_FROM_REMILL
+#include <remill/Arch/PPC/Runtime/State.h>
+
+namespace remill {
+
+namespace sleighppc {
+
+static const char *kPPCVLERegName = "VLEReg";
```

Comment:
Can be `constexpr`.

Reply:
[0f8ec93](https://github.com/lifting-bits/remill/pull/645/commits/0f8ec9393382a33c6af520851652da7ec90b1af6)